ext/include/opentelemetry/ext/zpages/threadsafe_span_data.h

```diff
@@ -175,8 +175,9 @@ class ThreadsafeSpanData final : public opentelemetry::sdk::trace::Recordable
   void AddEvent(
       nostd::string_view name,
       core::SystemTimestamp timestamp = core::SystemTimestamp(std::chrono::system_clock::now()),
-      const trace_api::KeyValueIterable &attributes =
-          trace_api::KeyValueIterableView<std::map<std::string, int>>({})) noexcept override
+      const opentelemetry::common::KeyValueIterable &attributes =
+          opentelemetry::common::KeyValueIterableView<std::map<std::string, int>>(
```

Comment:
With this change, can we get use of the `namespace trace_api = opentelemetry::trace;` at the top of this file?

Reply:
Did you mean to get remove the line `namespace trace_api = opentelemetry::trace;`? I think trace_api:: is still being used by`trace_api::CanonicalCode` in line 140. Would you prefer it be removed and use `opentelemetry::trace::CanonicalCode` instead?

Reply:
Yes, I think we can remove it if it's just one occurrence.